Project/Src/Gui/GutterMargin.cs

```diff
@@ -71,12 +74,17 @@ public override void Paint(Graphics g, Rectangle rect)
                 {
                     g.FillRectangle(fillBrush, backgroundRectangle);
                     var curLine = textArea.Document.GetFirstLogicalLine(textArea.Document.GetVisibleLine(textArea.TextView.FirstVisibleLine) + y);
+                    bool isCurrentLine = curLine == textArea.Caret.Line && MarkSelectedLine;
+                    Brush lineBrush = isCurrentLine ? currentLineBrush : drawBrush;
+                    Font font = isCurrentLine
+                        ? lineNumberCurrentPainterColor.GetFont(TextEditorProperties.FontContainer)
+                        : lineNumberPainterColor.GetFont(TextEditorProperties.FontContainer);
```

Comment:
Should these fonts be cached like the brushes?

Reply:
These are basically property getters, the only reason to cache font (or HighlightColor object) is for readability (same in the main PR for diffmargin).